packages/api-explorer/src/Doc.jsx

```diff
@@ -352,7 +528,7 @@ class Doc extends React.Component {
       if (lazy) {
         return (
           <Waypoint bottomOffset="-1%" fireOnRapidScroll={false} onEnter={this.waypointEntered}>
-            {this.state.showEndpoint && this.renderEndpoint()}
+            {this.state.showEndpoint ? this.renderEndpoint() : <div />}
```

Comment:
In test environments `react-waypoint` really doesn't like when we pass a bool as a child.